access-mgmt/README.adoc

```diff
@@ -0,0 +1,471 @@
+
+//URL components for cross refs
+:base-url: https://github.com/redhat-developer/app-services-guides/blob/main/
+:base-url-cli: https://github.com/redhat-developer/app-services-cli/tree/main/docs/
+:getting-started-url: getting-started/README.adoc
+:kafka-bin-scripts-url: kafka-bin-scripts/README.adoc
+:kafkacat-url: kafkacat/README.adoc
+:quarkus-url: quarkus/README.adoc
+:rhoas-cli-url: rhoas-cli/README.adoc
+:rhoas-cli-ref-url: commands
+:topic-config-url: topic-configuration/README.adoc
+:consumer-config-url: consumer-configuration/README.adoc
+:service-binding-url: service-discovery/README.adoc
+:access-mgmt-url: access-mgmt/README.adoc
+
+////
+END GENERATED ATTRIBUTES
+////
+
+[id="chap-managing-access"]
+= Managing account access in {product-long}
+ifdef::context[:parent-context: {context}]
+:context: managing-access
+
+[IMPORTANT]
+====
+{product-long} is currently available for Development Preview. Development Preview releases provide early access to a limited set of features that might not be fully tested and that might change in the final GA version. Users should not use Development Preview software in production or for business-critical workloads. Limited documentation is available for Development Preview releases and is typically focused on fundamental user goals.
+====
+
+// Purpose statement for the assembly
+[role="_abstract"]
+As an owner of a Kafka instance in {product}, you can manage the level of access that other user accounts and service accounts have to your instance. You can allow or deny access to your instance for specific accounts or for all accounts in your organization. You can also allow other users or service accounts to manage the level of access to your instance for you.
+
+You can manage access for only the Kafka instances that you create or for instances that the owner has enabled you to access and alter.
+
+//Additional line break to resolve mod docs generation error, not sure why. Leaving for now. (Stetson, 20 May 2021)
+
+[id="con-access-management_{context}"]
+== Access management in {product}
+
+[role="_abstract"]
+{product} uses Access Control Lists (ACLs) provided by Kafka that enable you to manage how other user accounts and service accounts are permitted to interact with the Kafka resources that you create. You can manage access for only the Kafka instances that you create or for instances that the owner has enabled you to access and alter.
+
+An account in {product} is either a user account or a service account. A user account enables users in your organization to access your resources. A service account enables your application or tool to connect securely to your instance and access your resources.
+
+A resource in an ACL can be a Kafka instance, topic, consumer group, or producer transaction. You use the ACL to define how specific accounts or all accounts in an organization are permitted to access these resources.
+
+You can define ACL permissions for the following components:
```

Comment:
This statement still seems off. These components you list are what make up one ACL. So something like this sounds more accurate:

> You can define ACL permissions using the following components:

or

> ACL permissions consist of the following properties: 

But I also am really unfamiliar with how to talk about ACLs, so take this with a grain of salt. I tried to find something comparable, and here's the [example from Confluent](https://docs.confluent.io/platform/current/kafka/authorization.html#acl-format), but it's not an exact match for how we're describing it here.

Reply:
Oh, I get it! Sorry, no idea why I wasn't getting that, and you're exactly right.

I've reworded as follows. Better?

>An ACL permission setting typically consists of the following components:

* A single named account or all accounts within the organization that you want to manage access for
* A single named resource, all resources of a particular type (such as a topic, consumer group, or transactional ID), or all resources of a particular type with a specified prefix
* A single operation (such as `Write`) or all operations for the specified resource or resources